jetbrains-core/src/software/aws/toolkits/jetbrains/core/stack/TreeView.kt

```diff
@@ -86,7 +86,7 @@ internal class TreeViewImpl(private val project: Project, stackName: String) : T
     }
 
     init {
-        val descriptor = StackNodeDescriptor(project, stackName, StatusType.UNKNOWN, message("cloudformation.stack.loading"))
+        val descriptor = StackNodeDescriptor(project, stackName, StatusType.UNKNOWN, message("loading_resource.loading"))
```

Comment:
Where else is this used?

Reply:
Since it wasn't deleted in the localized file

Reply:
fixed - not sure why IDEA wasn't showing it us unsed.